app/src/main/java/org/mozilla/fenix/components/Analytics.kt

```diff
@@ -69,12 +70,16 @@ class Analytics(
         val intent = Intent(context, HomeActivity::class.java).apply {
             flags = Intent.FLAG_ACTIVITY_NEW_TASK or Intent.FLAG_ACTIVITY_CLEAR_TOP
         }
-
+        val crashReporterIntentFlag = if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.S) {
+                PendingIntent.FLAG_MUTABLE
+        } else {
+            0 // No flags. Default behavior.
+        }
```

Comment:
Could we use `IntentUtils.defaultIntentPendingFlags` here? :)

Reply:
@Amejia481 No, since it contains FLAG_IMMUTABLE and here we require FLAG_MUTABLE in case a non fatal crash happens, this gives the app the opportunity to show an in-app confirmation UI before sending a crash report.

Reply:
OK, got it thanks for the clarification :)